README.rst

```diff
@@ -32,6 +32,14 @@ to support modern computational graph libraries such as PyTorch and
 TensorFlow in order to make use of features such as autodifferentiation
 and GPU acceleration.
 
+User Guide
+----------
+
+For an in depth walkthrough of usage of the latest release of ``pyhf`` visit the |pyhf tutorial|_.
```

Comment:
I was wondering whether it also makes sense to directly link the readthedocs here? The tutorial does so in the first line. It is also in the questions section in the readme as well as in the badges, but both of those are slightly more hidden compared to this new entry. A user will definitely find it regardless, but the benefit I see from also linking the RTD here is that they can be contrasted a bit so a user knows which website is more appropriate for their specific needs.

Reply:
I think that's a good idea. I put it lower down so that a reader would be hit first by the user guide, then by examples, and then by the docs. Very happy for other thoughts here though. :+1:

Reply:
No strong opinion here, I think it is fine as-is. I personally think that the "What does it support" section is expert-level material, and that there is a lot of scrolling needed to get to the docs section, but ultimately it is just personal opinion. Users should be able to comfortably find what they are looking for with the layout implemented here.

Reply:
I think I agree with you here. :+1: As that will take more thought then I think I have in me now, I've bumped that to Issue #1791.